packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -8,9 +8,9 @@
 #import <AVFoundation/AVFoundation.h>
 #import <GLKit/GLKit.h>
 
-#import "AVAssetTrackUtils.h"
-#import "FVPDisplayLink.h"
-#import "messages.g.h"
+#import "./include/video_player_avfoundation/AVAssetTrackUtils.h"
+#import "./include/video_player_avfoundation/FVPDisplayLink.h"
+#import "./include/video_player_avfoundation/messages.g.h"
```

Comment:
Do we know why include paths aren't taking care of this?

Reply:
So for most of the headers, include paths is sufficient. However, `messages.g.h` is a problem when there's multiple plugins that also have a `messages.g.h`. Seems like it can't resolve which to use. More explanation here: https://github.com/flutter/flutter/issues/147587#issuecomment-2086461349

So I decided to change all of them to relative paths to keep it uniform